ts/dialog/actions.ts

```diff
@@ -67,22 +67,14 @@ export const parseActions = (options: ConfirmationOptions, componentId?: string)
     }, options.accept)
   }
 
-  return Object.assign({}, ...['accept', 'reject'].map(actionName => {
+  return Object.assign({}, ...['accept', 'reject', 'close'].map(actionName => {
     const action = Object.assign({}, options[actionName]) as ActionOptions
     action.label = getActionLabel(options, action, actionName)
 
     if (!action.execute) {
       action.execute = parseAction(action, componentId)
     }
 
-    if (
-      actionName === 'accept'
-      && !action.color
-      && typeof options.icon === 'string'
-    ) {
-      action.color = iconsMap[options.icon] ?? options.icon
-    }
-
```

Comment:
I think that you can rollback this

Reply:
Why did you remove it?

Reply:
i remove this line because it will not rendered the icon. lets say i want to use "user" as icon, it will give u error action color = user.

Reply:
I understand, but it removes the default color.
You can fix it by applying another verification, checking if the icon is one of the default icons
`['success', 'error', 'info', 'warning', 'question']`
Check if is a key of the `icons` variable at `ts/dialog/icons.ts`